### ISSUE TITLE
Youth/Community signups [migrated from old Collection repo]

### ISSUE THREAD

**Issue by jennpb** [wellcometrust/wellcomecollection.org#826]
@davidmillerwellcome commented on [Mon May 09 2016](https://github.com/wellcometrust/collection/issues/153)

As PO, I want to raise the visibility of the Young people, teachers and community workers form so that more users sign up

## Description/vision/rationale

This follows the work in #42 (merge x3 forms / improve the design of forms).  Claire Carlin/Russell suggested that we think about promoting the newsletter throughout the Youth section.  We have the general WC sign-up in the footer, so it would be nice to have the equivalent promotion for Youth in that section.

David has given consideration to its general position.  Design should now take this ticket forward.
## Q&A

NA
## Acceptance criteria

Design/FE/BE TBC
## Related/dependencies
#42 As UX, I want to have the Youth & schools forms properly designed and integrated into the site so they are easy to use (this ticket depends on #42 because it needs a single form whereas currently we have three forms)
## Sketches/mockups/screenshots

Mocked up by David.  See [comment below for details](https://github.com/wellcometrust/collection/issues/153#issuecomment-221206789).  Potential idea showing:
i) link in left navigation
ii) general position
**Note:** Design now need to take this forward.

![image](https://cloud.githubusercontent.com/assets/14291978/15498043/8de5c0a0-2195-11e6-9e64-0bec01c4ef5e.png)

---

@davidmillerwellcome commented on [Mon May 09 2016](https://github.com/wellcometrust/collection/issues/153#issuecomment-217854397)

fyi @debdoop & @RussellDornan.  I'll have a think about this one in this sprint too. 

---

@davidmillerwellcome commented on [Tue May 24 2016](https://github.com/wellcometrust/collection/issues/153#issuecomment-221206054)

Blocked by: 
#42 In order to promote a single form throughout the section and orientate users, as UX, I want the x3 Youth section forms to be merged/redesigned/updated to reflect business needs 

---

@davidmillerwellcome commented on [Tue May 24 2016](https://github.com/wellcometrust/collection/issues/153#issuecomment-221206789)

To @debdoop & @RussellDornan on slack the other day (no need to reply, just want to record the progress)

> In the forms meeting the other day we talking about trying to promote the form throughout the section.  I had a wee think, and recommend adding it to the bottom, throughout the section.  I thought about top right, but I don’t think that’s going to work
> 
> I thought we could have something that spreads across the page rather than the standard block as per the rest of the site (although you could still go down that route if you really wanted to).  Perhaps Josh (digital designer) will have some other ideas in terms of layout.
> 
> But I do think the bottom of the page is the best location for it
> 
> In the mockup below, I haven’t attempted to style this - its just to show a general position.
> 
> I’ve also added a link to to sign up in the left nav

![image](https://cloud.githubusercontent.com/assets/14291978/15498030/7f08686c-2195-11e6-9dc4-41d4f2fd3ae7.png)
